astropy/modeling/separable.py

```diff
@@ -143,8 +142,9 @@ def _arith_oper(left, right):
     result : ndarray
         Result from this operation.
     """
-    # models have the same number of inputs and outputs
+
     def _n_inputs_outputs(input):
+        """Models have the same number of inputs and outputs."""
```

Comment:
@WilliamJamieson , is this where that comment was supposed to go?

Reply:
Looking at this more closely, this isn't quite the correct fix either. I'll push a fix directly.